ui/src/timeMachine/actions/index.ts

```diff
@@ -100,6 +100,11 @@ export type Action =
   | ReturnType<typeof updateCheckThreshold>
   | ReturnType<typeof removeCheckThreshold>
 
+type ExternalActions =
```

Comment:
ooh love this! Maybe we could even import and export all Actions from a central place?

Reply:
Yes, I want to do that in another PR.